unstructured/partition/text.py

```diff
@@ -166,6 +167,7 @@ def partition_text(
     paragraph_grouper: Optional[Callable[[str], str]] = None,
     metadata_filename: Optional[str] = None,
     include_metadata: bool = True,
+    languages: List[str] = ["auto"],
```

Comment:
should this typing be Optional[List[str]]?

Reply:
I don't think so - we are using a default "auto" value, so any other non-default value passed in by a user should also not be None. Let me know if that makes sense!